app/views/support/frameworks.html.haml

```diff
@@ -1,29 +1,9 @@
 .govuk-grid-row
-  .govuk-grid-column-full
-    %h1.govuk-heading-xl
+  .govuk-grid-column-two-third
+    %h1.govuk-heading-l
       Check which service to report to
-.govuk-grid-row
-  .govuk-grid-column-two-thirds
-    %p
-      The following frameworks are submitted through this service.
+
     %p
-      If you are reporting on any other framework, use the existing
-      = succeed '.' do
-        = link_to 'MISO service', 'https://miso.ccs.cabinetoffice.gov.uk/', :title => 'Link to MISO Service'
+      As of March 2020, all frameworks should be reported using this service.
```

Comment:
NB: This page is now inaccessible having removed the link from the guest login page. Keeping this here in case anyone has bookmarked it, to avoid more 404 errors cropping up in Rollbar.